### ISSUE TITLE
CSS Parts

### ISSUE THREAD

**Issue by dliebner** [github/image-crop-element#44]
Any thoughts on adding some [part](https://developer.mozilla.org/en-US/docs/Web/CSS/::part markup to make the component more styleable?

**Comment by keithamus** [comment]
Sounds great! PRs welcome!

**Comment by koddsson** [comment]
Yeah, being able to round the crop window with CSS Parts sounds good. We could then start deprecating the `rounded` attribute (or keep it for convenience). Developers can then change the shape of the crop window however they'd like.
